### ISSUE TITLE
Allow .from_csv to accept URL

### ISSUE THREAD

**Issue by v0dro** [SciRuby/daru#160]
The `DataFrame.from_csv` method only accepts file paths currently. Extend it to accept URLs.

**Comment by zverok** [comment]
Just found a pretty cool thing that could be in help somehow: https://github.com/seamusabshere/remote_table

**Comment by Shekharrajak** [comment]
We can use `open-uri` 

```
content = open('link/file.csv')
File.write('file.csv', content.read)
```
and then we can use `file.csv`, 

```
df = Daru::DataFrame.from_csv "file.csv"
```
right ?

**Comment by zverok** [comment]
@Shekharrajak This issue is already worked on https://github.com/SciRuby/daru/pull/300 -- though, the last movements was 2 weeks ago... So, I am not sure.
@anshuman23, do you plan to finalize your PR or what?..

**Comment by Shekharrajak** [comment]
Thanks for info.

**Comment by anshuman23** [comment]
@zverok - Sorry for not informing you earlier, but it completely slipped my mind. My college exams had started just a day after I last committed changes. However, they get over by Monday the coming week. So I will finalize the PR by Tuesday or at most Wednesday positively. Most of the requested changes are almost ready on my end.

**Comment by athityakumar** [comment]
I think this issue can be closed, as this has been fixed by PR #334.
